### ISSUE TITLE
[FYI] I made a demo request at livekits's agents to showcase Moshi

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [kyutai-labs/moshi#132]
### Due diligence

- [X] I have done my due diligence in trying to find the answer myself.

### Topic

Other / All

### Question

Hi,

I saw [on hackernews](https://news.ycombinator.com/item?id=41743327) that [agents by livekits used to make](https://github.com/livekit/agents) the openai realtime api as well as [cerebras voice](https://cerebras.vercel.app/) seems to be open source and ehey have tons of demos and code [on their github](https://github.com/livekit/agents?tab=readme-ov-file).

But they seem to lack demos for LLM that can do STT+TTS, so FYI I made a [an issue requesting a demo for Moshi using their SDK on their github](https://github.com/livekit/agents/issues/844) and am notifying you if you want to help them make it happen.

Feel free to close this.

Thank you very much.

**Comment by davidzhao** [comment]
We'd love to integrate Moshi with Agents! A few weeks ago we opened a PR [here](https://github.com/kyutai-labs/moshi/pull/7) with a proof of concept integration. It'd be great to get that merged.

**Comment by LaurentMazare** [comment]
Right, as mentioned offline to @theomonnom  we would certainly be happy to get this integrated so he could either re-open the PR (there might be a couple tweaks to do since the release, e.g. the repo names) or make a new one.

**Comment by thiswillbeyourgithub** [comment]
Great to here. Also I stumbled upon [this integration](https://github.com/modal-labs/quillman) using modal.com, which might be interesting go you.

**Comment by LaurentMazare** [comment]
Oh wow, thanks for pointing it out, I don't think any of us noticed the modal integration, it's very neat!
